unit-tests/run-unit-tests.py

```diff
@@ -503,10 +506,11 @@ def devices_by_test_config( test ):
 def test_wrapper( test, configuration = None ):
     global n_tests
     n_tests += 1
-    if configuration:
-        log.progress( '[' + ' '.join( configuration ) + ']', test.name, '...' )
-    else:
-        log.progress( test.name, '...' )
+    if not log.is_debug_on()  or  log.is_color_on():

```

Comment:
Why add the color check?
Not just debug check?

Reply:
If color is on, then the terminal supports advanced controls and progress will not stay on-screen. I wanted to avoid the progress since debug includes the same information, for readability on the libci logs. But when I run it locally, I run with debug and I have color, so it still shows progress...